Sources/MapboxNavigation/NavigationView.swift

```diff
@@ -196,6 +196,7 @@ open class NavigationView: UIView {
     public init(frame: CGRect,
                 tileStoreLocation: TileStoreConfiguration.Location? = .default,
                 navigationMapView: NavigationMapView? = nil) {
+        navigationMapView?.translatesAutoresizingMaskIntoConstraints = false
```

Comment:
It's also worth noting that if client sets `translatesAutoresizingMaskIntoConstraints` to `false` this issue won't happen.

I also see some potential issues with `didSet` in `NavigationView.navigationMapView`, since it'll not be called in initializers.

It's good that you've fixed original issue, but we'll have to do some more testing in the wild to see what else might be wrong. For that I think having `NavigationMapView` injection in our developer example would be good to have.